tests/decorators/test_decorators.py

```diff
@@ -0,0 +1,802 @@
+                                     }]})
+
+    def test_service_bus_queue_default_args(self):
+        app = self.func_app
+
+        @app.on_service_bus_queue_change(arg_name="req",
+                                         connection="dummy_conn",
+                                         queue_name="dummy_queue")
+        @app.write_service_bus_queue(arg_name='res',
+                                     connection='dummy_out_conn',
+                                     queue_name='dummy_out_queue')
+        def dummy():
+            pass
+
+        func = get_func(self, app)
+
+        assert_json(self, func, {"scriptFile": "function_app.py",
+                                 "bindings": [
+                                     {
+                                         "direction": BindingDirection.OUT,
+                                         "type": "serviceBus",
+                                         "name": "res",
+                                         "connection": "dummy_out_conn",
+                                         "queueName": "dummy_out_queue"
+                                     },
+                                     {
+                                         "direction": BindingDirection.IN,
+                                         "type": "serviceBusTrigger",
+                                         "name": "req",
+                                         "connection": "dummy_conn",
+                                         "queueName": "dummy_queue"
+                                     }
+                                 ]
+                                 })
+
+    def test_service_bus_queue_full_args(self):
+        app = self.func_app
+
+        @app.on_service_bus_queue_change(arg_name="req",
+                                         connection="dummy_conn",
+                                         queue_name="dummy_queue",
+                                         data_type=DataType.STREAM,
+                                         access_rights=AccessRights.MANAGE,
+                                         is_sessions_enabled=True,
+                                         cardinality=Cardinality.MANY)
+        @app.write_service_bus_queue(arg_name='res',
+                                     connection='dummy_out_conn',
+                                     queue_name='dummy_out_queue',
+                                     data_type=DataType.STREAM,
+                                     access_rights=AccessRights.MANAGE)
+        def dummy():
+            pass
+
+        func = get_func(self, app)
+
+        assert_json(self, func, {"scriptFile": "function_app.py",
+                                 "bindings": [
+                                     {
+                                         "direction": BindingDirection.OUT,
+                                         "dataType": DataType.STREAM,
+                                         "type": "serviceBus",
```

Comment:
might as well use the Constants defined for Trigger and bindings name - it'll validate that as well and prevent any typos in the test.